block/block_service.proto

```diff
@@ -595,6 +607,12 @@ enum SubscribeStreamResponseCode {
      * `end_block_number` field to a valid end block.
      */
     READ_STREAM_INVALID_END_BLOCK_NUMBER = 4;
+
+    /**
+     * The requested stream is not available.<br/>
+     * The client MAY retry again later.
+     */
+    READ_STREAM_NOT_AVAILABLE = 5;
```

Comment:
`READ_` is implied by 'subscription' imo, but not a big deal

Reply:
Makes sense. I added the READ_ prefix to align with others in `SubscribeStreamResponseCode`